AreaFiftyLAN/lancie-frontend
# 2.0 preview

This makes the frontend homepage work in Polymer 2. There are some things I found out:

- Activity dialog is broken
- Gold email input does not have a 2.0 version, can't we just use `<input type="email">`?
- There are some elements which have horrible code. Not sure if these are salvagable or that we just rewrite them.

To test this branch:

```
rm -rf bower_components/
bower install
yarn run serve
```

You can check the amount of linter warnings with `polymer lint`. We still have some related to `iron-lazy-pages`, I will need to take a look there.
